.github/workflows/issue-management.yml

```diff
@@ -0,0 +1,21 @@
+name: Issue management
+
+on:
+  schedule:
+  - cron: '41 8 * * *'
+
+jobs:
+  stale:
+    name: Close stale awaiting-feedback issues
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        only-labels: awaiting-feedback
+        stale-issue-message: 'Since we haven''t heard back from you, closing this issue for now. Please feel free to comment with more information and we can get this reopened.'
```

Comment:
Should we also start adopting `stale-issue-label` to better track which issues get auto-closed out?

Reply:
Ah, that's probably a good idea. Perhaps `awaiting-feedback-stale`?